symfony/plugins/orangehrmAttendancePlugin/config/routes.yaml

```diff
@@ -0,0 +1,48 @@
+apiv2_attendance_configure:
+  path: /api/v2/attendance/configure
+  controller: OrangeHRM\Attendance\Controller\AttendanceMockController::getAttendanceConfig
+  methods: [ GET ]
+
+apiv2_save_attendance_configure:
+  path: /api/v2/attendance/configure
+  controller: OrangeHRM\Attendance\Controller\AttendanceMockController::setAttendanceConfig
+  methods: [ PUT ]
+
+apiv2_attendance_currentdatetime:
+  path: /api/v2/attendance/currentdatetime
+  controller: OrangeHRM\Attendance\Controller\AttendanceMockController::getCurrentDateTime
+  methods: [ GET ]
+
+apiv2_attendance_validate_datetime:
+  path: /api/v2/attendance/validatedate
+  controller: OrangeHRM\Attendance\Controller\AttendanceMockController::validateDateTime
+  methods: [ GET ]
+
+apiv2_attendance_records:
+  path: /api/v2/attendance/records/{id?}
```

Comment:
is this `?` in `{id?}` for regex?

Reply:
Since this route is mock i just added GET/PUT/POST all into one method and i needed to match it with or without an `id`. followed [optional parameter](https://symfony.com/doc/current/routing.html#optional-parameters) docs